File: zad1&2/Tipper/Tipper/app/src/main/java/com/example/tipper/activities/QuizQuestion.java
package com.example.tipper.activities;

public class QuizQuestion {

    private String question;
    private String answer1;
    private String answer2;
    private int goodAnswerIndex;

    public int getGoodAnswerIndex() {
        return goodAnswerIndex;
    }

    public String getQuestion() {
        return question;
    }

    public String getAnswer1() {
        return answer1;
    }

    public String getAnswer2() {
        return answer2;
    }

    public QuizQuestion(String question, String answer1, String answer2, int goodAnswerIndex) {
        this.question = question;
        this.answer1 = answer1;
        this.answer2 = answer2;
        this.goodAnswerIndex = goodAnswerIndex;
    }
}

File: zad1&2/Tipper/Tipper/app/src/main/java/com/example/tipper/activities/QuizQuestion.kt
package com.example.tipper.activities

class QuizQuestion(private val question: String?, private val answer1: String?, private val answer2: String?, private val goodAnswerIndex: Int) {
    fun getGoodAnswerIndex(): Int {
        return goodAnswerIndex
    }

    fun getQuestion(): String? {
        return question
    }

    fun getAnswer1(): String? {
        return answer1
    }

    fun getAnswer2(): String? {
        return answer2
    }
}